crates/misc/tct-visualize/src/bin/tct-live-edit.rs

```diff
@@ -79,8 +80,8 @@ async fn main() -> anyhow::Result<()> {
                 .unwrap()
         }
         (None, None) => {
-            axum::Server::bind(&address)
-                .serve(app.into_make_service())
+            let listener = tokio::net::TcpListener::bind(&address).await.unwrap();
```

Comment:
This could be seen also in other parts of the penumbra stack: axum has removed its `Server` type, instead replacing it with a `axum::serve` utiilty function.